TESTS/usb_device/serial/main.cpp

```diff
@@ -736,19 +766,30 @@ void test_serial_printf_scanf()
     }
     // Wait for the host to close its port.
     while (usb_serial.ready()) {
-        ThisThread::sleep_for(1);
+        ThisThread::sleep_for(1ms);
     }
     usb_serial.disconnect();
 }
 
 void line_coding_changed_cb(int baud, int bits, int parity, int stop)
 {
+#if defined(MBED_CONF_RTOS_PRESENT)
     line_coding_t *lc = lc_mail.alloc();
```

Comment:
Is there any advantage to keeping the RTOS implementation around? The CircularBuffer implementation should be able to handle both cases right? If so you could remove this.